WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2738,6 +2737,14 @@ static GSourceFuncs _handlerIntervention =
             }
 
             _adminLock.Unlock();
+
+            TRACE_L1("%s and _httpStatusCode : %d ", URL.c_str(), _httpStatusCode);
+            if ( (_httpStatusCode != 200 ) && (_httpStatusCode != -1)) {
+                notifyUrlLoadResult(URL, Core::ERROR_INCORRECT_URL);
+            }
+            else {
+                notifyUrlLoadResult(URL, Core::ERROR_NONE);
+            }
```

Comment:
I prefer to have a single call to notifyUrlLoadResult() function, so it would be better to set just a status code.

             uint32_t status = Core::ERROR_NONE;
             TRACE_L1("URL: %s, _httpStatusCode : %d ", URL.c_str(), _httpStatusCode);
  
             if ((_httpStatusCode != 200 ) && (_httpStatusCode != -1)) {
                 status = Core::ERROR_INCORRECT_URL;
             }
             notifyUrlLoadResult(URL, status);

Reply:
yes, I will move this to up(before client notification)